tests/testFootprintMergeCatalog.py

```diff
@@ -122,23 +140,22 @@ def testMerge1(self):
         pixArea.fill(69)
         pixArea[1] = 135
         measArea = [i.getFootprint().getArea() for i in merge]
-        self.assert_(np.all(pixArea == measArea))
+        np.testing.assert_array_equal(pixArea, measArea)
```

Comment:
utils.assert* don't work on lists or tuples (it's documented as "array-like"). The numpy methods do work on other generic sequences, so this seems fine to me here. The other option would be to do `np.array(pixArea)` inside one of our tests.